components/content-service/pkg/git/git.go

```diff
@@ -169,6 +169,14 @@ func (c *Client) GitWithOutput(ctx context.Context, subcommand string, args ...s
 		env = append(env, fmt.Sprintf("GIT_AUTH_PASSWORD=%s", pwd))
 	}
 
+	homeDir, err := os.MkdirTemp("", "git")
+	if err != nil {
+		return nil, xerrors.Errorf("cannot create temporal directory")
+	}
+	defer os.RemoveAll(homeDir)
+
+	env = append(env, fmt.Sprintf("HOME=%v", homeDir))
```

Comment:
Hm, I'm not sure I understand. I thought:
 - git would store global config in `~/.gitconfig`
 - if we do not have a shared `HOME` folder across all commands, how is `safe.directory` picked up here? :thinking:

Reply:
it's just needed for the git status. Doesn't matter if this config persists, so long as the commands during the git status collection pick it up.